modules/reindex/src/main/java/org/elasticsearch/index/reindex/BulkByScrollParallelizationHelper.java

```diff
@@ -154,19 +154,9 @@ private static int countSlicesBasedOnShards(ClusterSearchShardsResponse response
                 }
                 slicedSource = request.source().copyWithNewSlice(sliceBuilder);
             }
-            slices[slice] = new SearchRequest()
-                    .source(slicedSource)
-                    .searchType(request.searchType())
-                    .indices(request.indices())
-                    .types(request.types())
-                    .routing(request.routing())
-                    .preference(request.preference())
-                    .requestCache(request.requestCache())
-                    .scroll(request.scroll())
-                    .indicesOptions(request.indicesOptions());
-            if (request.allowPartialSearchResults() != null) {
-                slices[slice].allowPartialSearchResults(request.allowPartialSearchResults());
-            }
```

Comment:
The following fields were not copied:

```
    this.batchedReduceSize = searchRequest.batchedReduceSize;
    this.maxConcurrentShardRequests = searchRequest.maxConcurrentShardRequests;
    this.preFilterShardSize = searchRequest.preFilterShardSize;
```

They look recently added hence I assume this was just a mistake and they should be copied.